includes/classTracksDB.php

```diff
@@ -0,0 +1,30 @@
+<?php
+require_once './includes/classSQLPlus.php';
+class TracksDB extends SQLPlus 
+{ 
+    
+    function __construct($params)
+    {
+        parent::__construct($params);
+    }
+ 
+    //*********************************************************************
+    // Tracks
+    //*********************************************************************  
+    public function numTracksForFlight($start,$end,$aircraft)
```

Comment:
Let me check to see where else its used in the helper.  If I cannot find anywhere else I will reaplce remove from helper also, as thats is very old code.

Reply:
It is still used in MyFlights and the igc generator.  
Best option will be to leave as buggy now, and I will patch the above 2 as a separye bug as I need to change the mapping with another branch anyway.

Reply:
Yes, it is still used but it can be replaced
e.g.
in igcgenerate.php:46
```
if (tracksforFlight($con,null,$glider,$trDateStart->format('Y-m-d H:i:s'),$trDateLand->format('Y-m-d H:i:s')) )
```
can be replaced by
```
if(GlidingDB->numTracksForFlight(....) > 0)
```
and on line 52
```
if (tracksforFlight(null,$con2,$glider,$trDateStart->format('Y-m-d H:i:s'),$trDateLand->format('Y-m-d H:i:s')) )
```
can be replaced by
```
if(TracksDB->numTracksForFlight(....) >0)
```


Am I wrong?

Reply:
You are correct.  Havn't tested by yes this woudl work.  Was going o be my next rewrite, but as igs=c is very raely used, was concentraing on the problems we had expereinced with archive first.
Tim

Reply:
Maybe I am missing something, but everywhere `tracksforFlight` is called, it is used like
```
if (tracksforFlight(....)) {
  either load somthing from gliding or archives database
}
```
so I think we can replace all calls to `itracksforFlight` with either a call to `GlidingDB->numTracksForFlight` or `TracksDB->numTracksForFlight` depending on which DB is referenced inside the `if` block (gliding or archives)

Reply:
Correct, but I havn't gone through them all with  double check yet.
I suggest I add to this branch the otherchnages and remove from helpers.
My point was to do that later, and simply hurry to get tracks working, but I will therfore delay and fix inother places.  Let you know when completed,

Reply:
> My point was to do that later, and simply hurry to get tracks working, but I will therfore delay and fix inother places. Let you know when completed,

I am sorry for the mess I have created with that buggy helper. 
If in a hurry then we can absolutely merge this branch and address the bug in another change.
Absolutely fine with me.